### ISSUE TITLE
Jitsi e MM: possibilità di bloccare riquadri utente

### ISSUE THREAD

**Issue by ivoermejo** [iorestoacasa-work/iorestoacasa.work#103]
sia con Jitsi che con MM
se devo registrare con OBS-Studio, o altri programmi, i relatori e i relativi desktop condivisi, si deve selezione la finestra e ritagliare il riquadro impostando la posizione esatta, per esempio, del relatore.
Ma se un utente entra o esce, la griglia degli utenti collegati cambia: si riordina, a volte si rimpiccioliscono i riquadri....
Proporrei un pulsante "pin" per bloccare i riquadri che si vogliono fissare.

**Comment by tapionx** [comment]
non puoi usare la modalità filmstrip, cosi si vede sempre grande in alto chi sta parlando in quel momento?

**Comment by ivoermejo** [comment]
no, perché spesso mi viene chiesto di effettuare il PIP, desktop in grande e relatore in piccolo.

**Comment by tapionx** [comment]
in Jitsi puoi premere su un partecipante per fissarlo e vederlo grande, invece in MM potresti "staccare" il relatore che ti interessa su una nuova finestra e mettere quella in OBS

**Comment by ivoermejo** [comment]
Jitsi se ho 3 relatori e 2 desktop da registrare, ovviamente in varie scene, non è congeniale.
In MM devo ancora provare l'opzione di staccare le finestre dei singoli relatori e desktop, e capire bene come impostare correttamente le fonti in OBS-Studio.
Comunque grazie.

**Comment by ivoermejo** [comment]
suggerimento, puoi provare ad usare https://obs.ninja/
da là puoi creare dei link, i quali li passerai ai relatori, per abilitare webcam e scrivania
il link che passi ai relatori sono "push" invece lo stesso link per usarlo in programmi come OBS o nel browser per vedere diventa "view"
il resto è solo come sta meglio a te usare queste finestre con webcam e relatore
puoi comporre una scena di OBS-Studio e passare la composizione a Jitsti....
